ROADMAP.md

```diff
@@ -0,0 +1,73 @@
+# Roadmap
+
+  * [Documentation](#documentation)
+  * [Demonstrator](#demonstrator)
+  * [Converters](#converters)
+  * [Grid Modeling](#grid-modeling)
+  * [Simulators](#simulators)
+  * [Data management](#data-management)
+  * [Vizualisation](#data-management)
+  * [High level services](#high-level-services)
+
+
+## Documentation
+- Functional documentation
+- User stories
+- More and more tutorials
+
+## Demonstrator
```

Comment:
- Create web-based demos packaged as docker images to show the world what is powsybl